lib/registries/addon/drafts/draft/-components/right-nav/template.hbs

```diff
@@ -20,6 +20,7 @@
     <Drafts::Draft::-Components::Register
         @draftManager={{@draftManager}}
         @onSubmitRedirect={{@onSubmitRedirect}}
+        @clickable={{@draftManager.currentUserIsAdmin}}
```

Comment:
I think `currentUserCanRegister` or something similar would be carry more meaning here than `clickable`.

Reply:
Just realized there's `@alias('draftManager.currentUserIsAdmin')` so I will use that 👀

Reply:
if only the admin can register, do you need this argument at all?

Reply:
Nope, not needed at all. I'll push the changes up to remove this argument